AccidentHere/ViewController.swift

```diff
@@ -48,14 +59,49 @@ class ViewController: UIViewController, CLLocationManagerDelegate{
         if (speed > 0) {
             if (speed > baseline) {
                 self.view.backgroundColor = UIColor.red
+                self.statusLabel.textColor = UIColor.white
+                self.motion.startAccelerometerUpdates()
             } else {
                 self.view.backgroundColor = UIColor(named: "backgroundColor")
+                statusLabel.textColor = UIColor.systemGray5
+                self.motion.stopAccelerometerUpdates()
             }
             
             speedLabel.text = (String(format: "%.0f", speed))
         } else {
             speedLabel.text = "0"
         }
     }
+    
+    func initAccelerometer(){
+        if motion.isAccelerometerAvailable {
+            motion.accelerometerUpdateInterval = 1.0 / 60.0
+            
+            let sensitivityValue = self.collisionSensitivity
+            let timer = Timer(fire: Date(), interval: (1.0/60.0), repeats: true, block: { (timer) in
+                if let data = self.motion.accelerometerData {
+                    let val_x = data.acceleration.x
+                    let val_y = data.acceleration.y
+                    let val_z = data.acceleration.z
+                    
+                    if val_x > sensitivityValue || val_y > sensitivityValue || val_z > sensitivityValue{
```

Comment:
```suggestion
                    if val_x > sensitivityValue || val_y > sensitivityValue || val_z > sensitivityValue {
```
[SwiftLint](https://github.com/realm/SwiftLint) 사용하면 좋았을 것 같아요👀
한 번 확인해보셔염 👍 [Swift Style Guide](https://google.github.io/swift/)

Reply:
꿀팁 감사합니다 😊